### ISSUE TITLE
question about dataset & code

### ISSUE THREAD

**Issue by MohammadHossein-esmaili** [AfsanaAhmedMunia/AHF-Fusion-U-Net#4]
Hello, I have reproduced your code and found that you only divide the data set into training set and validation set. Where is the test set?
Did you use the same evaluation data for the final test section for dice and ...? 
I ran it and didn't get the results announced
@AfsanaAhmedMunia @moloud1987
